tailcfg/tailcfg.go

```diff
@@ -2342,6 +2342,10 @@ const (
 	// NodeAttrDisableCaptivePortalDetection instructs the client to not perform captive portal detection
 	// automatically when the network state changes.
 	NodeAttrDisableCaptivePortalDetection NodeCapability = "disable-captive-portal-detection"
+
+	// NodeAttrSSHEnvironmentVariables enables logic for handling environment variables sent
+	// via SendEnv in the SSH server and applying them to the SSH session.
+	NodeAttrSSHEnvironmentVariables NodeCapability = "ssh-environment-variables"
```

Comment:
I guess this verbose naming is consistent with some of the ones above, but those are all rare/debug things that don't normally get sent. For things appearing regularly for normal users, I might err on shorter to save bytes on the wire, like `"ssh-env-vars"` but probably doesn't matter much.

Reply:
Sounds good, will adjust.